systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -395,6 +395,15 @@ DoneableKafka kafka(Kafka kafka) {
         });
     }
 
+    public Kafka deployUnsupportedKafka(Kafka kafka) {
+        kafka().inNamespace(client().getNamespace()).createOrReplace(kafka);
+        return kafka;
+    }
+
+    public void deleteUnsupportedKafka(Kafka kafka) {
+        kafka().inNamespace(client().getNamespace()).delete(kafka);
+    }
+
```

Comment:
The above methods just create/update and deploy a `Kafka` instance why the names reflect an "UnsopportedKafka"? They are generic enough for any valid `Kafka`.

Reply:
Ye that's correct. The thing is, I didn't want to use our normal kafka deployment which contains waits for SS, deplyments and pods. This is used only for specific use case. However, it make sense to change the name and just put there some note for which use cases it should be used.